package.json

```diff
@@ -230,6 +231,15 @@
           "@fluentui/react-conformance",
           "stylis"
         ]
+      },
+      {
+        "packages": [
+          "@fluentui/react-northstar-emotion-renderer"
+        ],
+        "dependencies": [
+          "@fluentui/eslint-plugin",
+          "stylis"
```

Comment:
As we are using Stylis v4, we can't use the same group as `@fluentui/react-northstar-fela-renderer` that uses Stylis v3...